### ISSUE TITLE
Using ffmpeg instead of flac

### ISSUE THREAD

**Issue by jcla1** [taf2/speech2text#6]
Hi there

I was trying out speech2text when I noticed that it used both ffmpeg and flac.

To make the lib more compatible with computers that haven't got flac installed, I suggest changing

```
lib/speech/audio_splitter.rb: line 46
```

to

``` ruby
chunk_outputfile = chunk.gsub(/#{File.extname(chunk)}$/, ".flac")
if system("ffmpeg -i #{chunk} -acodec flac #{chunk_outputfile}")
  [...]
```

I have tried it and it works fine on my laptop.

**Comment by taf2** [comment]
Great idea!   Thanks.

**Comment by jcla1** [comment]
I already have it coded. Shall I send a pull request?

**Comment by taf2** [comment]
sure, I'll merge

**Comment by jcla1** [comment]
This was resolved in #7.
